documentation/api/metrics/v1/mbeans.markdown

```diff
@@ -152,4 +152,43 @@ resources, storing edges, etc.). Metrics of particular note include:
 
 [Using `curl` from localhost][curl]:
 
+    curl 'http://localhost:8080/metrics/v1/mbeans/puppetlabs.puppetdb.command%3Atype%3Dglobal%2Cname%3Dprocessing-time'
```

Comment:
Since we're just presenting  this literally, I might be tempted to include the prompt do help distinguish command and response, i.e.

    $ curl ...
    response...

    $ curl ...
    response...